signup.html

```diff
@@ -0,0 +1,135 @@
+            <li><a href="#" class="nav__link">Adopt</a></li>
+            <li><a href="#" class="nav__link">Shelters</a></li>
+            <li><a href="#" class="nav__link">Donate</a></li>
+          </ul>
+        </div>
+        <div class="nav__icons">
+          <i class="material-icons">search</i>
+          <i class="material-icons">notifications</i>
+          <i class="material-icons">account_circle</i>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <div class="main__wrapper">
+        <div class="login__menu">
+          <button class="login__menu-type active" id="seeker" onclick="seeker()">
+            <h6>Seeker</h6>
+          </button>
+          <button class="login__menu-type" id="shelter" onclick="shelter()">
+            <h6>Shelter</h6>
+          </button>
+          <script>
+            var se = document.getElementById("seeker");
+            var sh = document.getElementById("shelter");
+    
+            function seeker(){
+              se.classList.add("active");
+              sh.classList.remove("active");
+            }
+            function shelter(){
+              se.classList.remove("active");
+              sh.classList.add("active");
+            }
+          </script>
+        </div>
+        <div class="login__wrapper">
+          <h5 class="login__form-title">Create an account</h5>
+          <!-- form does not submit anything anywhere for now -->
+          <form action="javascript:void(0);" method="post" class="login__form">
+            <div class="signup__form-field">
+              <label for="username">Username</label>
+              <input type="text" name="username" id="username" required>
+            </div>
+            <div class="signup__form-field">
+              <label for="firstname">First Name</label>
+              <input type="text" name="firstname" id="firstname" required>
+            </div>
+            <div class="signup__form-field">
+              <label for="lastname">Last Name</label>
+              <input type="text" name="lastname" id="lastname" required>
+            </div>
+            <div class="signup__form-field">
+              <label for="singup__email">Email</label>
+              <input type="email" name="singup__email" id="signup__email" required>
+            </div>
+            <div class="signup__form-field">
+              <label for="signup__pwd">Password</label>
+              <input type="password" name="signup__pwd" id="signup__pwd" required>
+            </div>
+            <div class="signup__form-field">
+              <label for="c__pwd">Confirm Password</label>
+              <input type="password" name="c__pwd" id="c__pwd" required>
+            </div>
+  
+            <!-- Use submit button insteaad of a tag once form becomes non-static -->
+            <!-- <button type="submit" value="Login" class="btn">Sign Up</button> -->
+            <a href="signup-success.html" class="btn">Sign Up</a>
```

Comment:
Form needs a submit button, I can directly access signup-success by clicking this. You can add JS code which checks the form's validity with checkValidity(), refer to my branch scripts/shelter.js if you want to look into it more. This could also reduce the the signup into only signup-success.html? Not sure what you want to do with signup-fail.html.

Reply:
Fixed in [force-push](https://github.com/JuneOh710/PetPal/compare/ed734821be957cb2b6c8cdca82520d86c44a8974..1da33d4db46bc811b42d34b6387a0acad9354278)